client/lib/menu-data/menu-data.js

```diff
@@ -349,10 +366,6 @@ MenuData.prototype.interceptLoadForHomepageLink = function ( menu ) {
 	return menu;
 };
 
-/**
- * @param {object} menu
- * @param {function} [optional] callback
- */
```

Comment:
I'm curious about why you deleted this jsdoc block. Is it because the meaning and parameters seem pretty obvious? I don't necessarily disagree, I just want to make sure I understand the reasoning.